tordnscrypt/src/main/res/values/strings.xml

```diff
@@ -153,113 +153,113 @@
     <string name="pref_fast_use_tor_bridges_add_dialog">Save</string>
     <string name="pref_fast_use_tor_bridges_close_dialog">Close</string>
-    <string name="pref_fast_unlock_host_wrong">Wrong host name!</string>
-    <string name="pref_fast_theme">Select Theme</string>
-    <string name="pref_fast_theme_titles_day_mode">Day Mode</string>
-    <string name="pref_fast_theme_titles_night_mode">Night Mode</string>
+    <string name="pref_fast_unlock_host_wrong">Wrong hostname!</string>
+    <string name="pref_fast_theme">Theme</string>
+    <string name="pref_fast_theme_titles_day_mode">Day</string>
+    <string name="pref_fast_theme_titles_night_mode">Night</string>
     <string name="pref_fast_theme_titles_auto_mode">Auto</string>
-    <string name="pref_fast_theme_titles_default_mode">System default</string>
+    <string name="pref_fast_theme_titles_default_mode">System</string>
     <string name="pref_fast_tor_bypass_lan">Bypass LAN addresses</string>
-    <string name="pref_fast_tor_bypass_lan_summ">Do not use Tor for LAN destinations and IANA reserved IP blocks</string>
-    <string name="pref_fast_language">Select Language</string>
-    <string name="pref_fast_block_http">Block http</string>
-    <string name="pref_fast_block_http_summ">Deny connection to http port 80</string>
-    <string name="pref_fast_categ_update">Update</string>
-    <string name="pref_fast_auto_update">Automatic Updates</string>
-    <string name="pref_fast_auto_update_summ">Check InviZible Pro and modules Updates daily</string>
-    <string name="pref_fast_chek_update">Check Update</string>
-    <string name="pref_fast_chek_update_summ">Check if new versions are available now</string>
-    <string name="pref_fast_through_tor_update">Update strictly through Tor</string>
-    <string name="pref_fast_through_tor_update_summ">Check updates only with Tor and when Tor is running</string>
-
-    <string name="pref_common_categ_run_with_root">Use Root Privileges</string>
+    <string name="pref_fast_tor_bypass_lan_summ">Do not use Tor for LAN destinations and IANA reserved IP blocks.</string>
+    <string name="pref_fast_language">Language</string>
+    <string name="pref_fast_block_http">Block HTTP</string>
+    <string name="pref_fast_block_http_summ">Deny HTTP connection over port 80.</string>
+    <string name="pref_fast_categ_update">New Versions</string>
+    <string name="pref_fast_auto_update">Auto-installation</string>
+    <string name="pref_fast_auto_update_summ">Check for new versions of InviZible Pro and modules daily.</string>
+    <string name="pref_fast_chek_update">Look for new versions</string>
+    <string name="pref_fast_chek_update_summ">Check if new versions are available now.</string>
+    <string name="pref_fast_through_tor_update">Download any new versions only through Tor</string>
+    <string name="pref_fast_through_tor_update_summ">Look for new versions only with Tor and when Tor is running.</string>
```

Comment:
This is not that good, and I don't understand how Tor running isn't a prerequisite for checking for new versions using it…